python/lsst/daf/butler/core/registry.py

```diff
@@ -168,7 +189,12 @@ def transaction(self):
 
     @property
     def pixelization(self):
-        """Object that interprets SkyPix DataUnit values (`sphgeom.Pixelization`)."""
+        """Object that interprets SkyPix DataUnit values (`sphgeom.Pixelization`).
+
+        ``None`` for limited registries.
```

Comment:
I think single back ticks work for `None` (not that anyone will really be clicking on it to find out what it means).